fe-test/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+
+networks:
+  development:
+    driver: bridge
+
+services:
+  app:
+    build:
+      context: ../
+      dockerfile: ../Dockerfile
+    command: sbt run
+    image: sbt
+    ports:
+      - "8080:8080"
+    volumes:
+      - "../:/root/build"
+    networks:
+      - development
+  tests:
+    build:
+      dockerfile: ./Dockerfile
+    command: npm run test
```

Comment:
doesn't like it: 
> OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "npm": executable file not found in $PATH: unknown

Ideally I like it to mirror this:

https://github.com/ShahOdin/dockerise-everything/blob/71f3107df6d1439d8aabdf0922a40e00ceb07503/run/docker-compose.yml#L4-L8

and not hard-code the npm run test in the docker file.